html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <form action="#" method="get" id="form">
+    <h1>Coding Form!</h1>
+    <div>
+      <label for="email">Email</label>
+      <div>
+        <input type="email" id="email" name="email">
+      </div>
+    </div>
+    <div>
+      <label for="why-is-coding-cool?">Why is coding cool?</label>
+      <div>
+        <select name="coding-reason" id="select">
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+    </div>
+    <div>
+      <p>Favorite Language (choose one)</p>
+      <label for="favorite-language"></label>
+      <div>
+        <input type="radio" id="html" name="language" value="html"> HTML
```

Comment:
make sure you have a label for each radio input, you can wrap them around the input if desired. That will maintain the same functionality